masonry/examples/calc_masonry.rs

```diff
@@ -176,7 +176,7 @@ impl Widget for CalcButton {
             match event.action {
                 accesskit::Action::Default => {
                     ctx.submit_action(Action::Other(Box::new(self.action)));
-                    ctx.request_paint();
+                    ctx.request_paint_only();
```

Comment:
I'm surprised this line is necessary at all. Nothing has changed which would require a repaint to happen.

Reply:
Right. Now that you mention it, the paint method of that widget is empty, so `request_paint` should never be called anyway.